site/app/views/grading/ElectronicGraderView.php

```diff
@@ -440,7 +440,7 @@ function sortTable(sort_element_index){
      * @param bool $show_edit_teams
      * @return string
      */
-    public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_users, $graders, $empty_teams, $show_all_sections_button, $show_import_teams_button, $show_export_teams_button, $show_edit_teams, $past_grade_start_date, $view_all, $sort, $direction, $anon_mode) {
+    public function detailsPage(Gradeable $gradeable, $graded_gradeables, $teamless_users, $graders, $empty_teams, $show_all_sections_button, $show_import_teams_button, $show_export_teams_button, $show_edit_teams, $past_grade_start_date, $view_all, $sort, $direction, $anon_mode, $overrides, $anon_ids) {
```

Comment:
Should type hints be added to all of these parameters as well?

Reply:
I didn't here cause there was just so many arguments. Not sure how good the type checking is here anyways cause it gets dynamically passed these.